### DISCUSSION TITLE
Speedometer movable

### DISCUSSION THREAD

**Discussion by Scrat-2023** [osmandapp/OsmAnd#20297] [Ideas]
Hello, in landscape mode the speedometer is hiding the left board.
Is it possible to make the speedometer moveable?
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/ee54f6a9-abf1-43e9-9961-bd12eea1d07a"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by dmpr0** [comment]
Related to https://github.com/osmandapp/OsmAnd/issues/20155. Unfortunately, now it's not possible
